### PR TITLE
Remove remaining hardcoded masks

### PR DESCRIPTION
**Pull request by phire**
Dolphin now supports overriding EXRAM size, so hardcoded masks will be incorrect

As for L1Cache, I don't think it ever makes sense to override it's size, but some of the masks were incorrect anyway.

### CODE REVIEW

File: Source/Core/Core/PowerPC/MMU.cpp [hunk 1]
```diff
@@ -320,7 +320,8 @@ static void WriteToHardware(u32 em_address, const T data)
       (em_address < (0xE0000000 + Memory::GetL1CacheSize())))
   {
     const T swapped_data = bswap(data);
-    std::memcpy(&Memory::m_pL1Cache[em_address & 0x0FFFFFFF], &swapped_data, sizeof(T));
+    std::memcpy(&Memory::m_pL1Cache[em_address & Memory::GetL1CacheMask()], &swapped_data,
```
Comment (BhaaLseN) [inline Source/Core/Core/PowerPC/MMU.cpp:323]: This changed from `0x0FFFFFFF` to `Memory::GetL1CacheMask()`, but every other occurrence used to be `0x3FFFF`. Is this correct, or should this be `Memory::GetExRamMask()` instead?

Comment (phire) [reply Source/Core/Core/PowerPC/MMU.cpp:323]: It certainly shouldn't be `GetExRamMask()`

Taking a better look at this code (and the code ExRam above), it doesn't actually implement mirroring, as the if statement is only matches for the first mirror.

All the `& 0x0fffffff` is doing is lopping off the `0x10000000` or `0xe000000`

**Comment by Tilka** [comment]
The file still mentions 0x0FFFFFFF a few times. Are all remaining occurrences meant to be static?

**Comment by phire** [comment]
I've decided against this cleanup

**Pull request closed without merging**